### PR TITLE
Add content to documentation

### LINKED ISSUE #2: Add content to Documentation
Currently the documentation is still a rough outline, inspired by the corresponding section of https://www.howtofair.dk. It needs to be developed further. Would be nice to use this figure: 

image

### PR DESCRIPTION
**Pull request by stefsmeets**
🚀 @Peter9192 🚀
Closes #2

### CODE REVIEW

File: _episodes/documentation.md [hunk 1]
```diff
@@ -1,33 +1,92 @@
...
+questions do they address? Was there any kind of quality assurrence?
 
-Project information: how was the data obtained (where there steps)? By whom? Are there different versions of the data? Quality assurance?
+## Where to document my data?
 
-On data specific info you often find difficult terms like ‘semantics’, ‘vocabularies’, ‘ontologies’, ‘linked data’. In climate (related) science, these are often so ‘logical’ that you won’t even notice they’re there. However, look at this example of a patient’s data model, and the netcdf model. Do you see it now? More info on this in sections file format and metadata.
+Documentation can take many forms: a data management plan, papers published,
+(jupyter) notebooks, lab journals, provenance information, etc. It is a good
+idea to bundle these with the data when you publish the data, so that people
+that find your data will not have to search for it elsewhere. Most data
```
Comment (nielsdrost) [inline _episodes/documentation.md]:
```suggestion
that find your data will not have to search for it elsewhere, and that access to documentation is guaranteed. Most data
```

File: _episodes/documentation.md [hunk 2]
```diff
@@ -1,33 +1,92 @@
...
+any information that is not covered by the other metadata fields.
 
-Project info is typically written in the data description fields, but may also include pointers to external documents, such as the data management plans.
+> ## Grouping related files with Zenodo communities
+> Data documentation in your project may be scattered over many files of
+> different types. To bundle this information, Zenodo provides 'communities'.
+> You can upload all your files to Zenodo and add them to a community. You can
+> also track versions. Each file gets its own DOI. So you could e.g. cite your
+> data management plan in the description field of your dataset.
+{: .callout}
```
Comment (nielsdrost) [inline _episodes/documentation.md]:
```suggestion
> Data documentation in your project may be scattered over many files of
> different types. To bundle this information, Zenodo allows you to upload
> a set of related files (hundreds if needed) as a single dataset.
> 
> In addition, Zenodo provides 'communities'. You can upload all your datasets
> to Zenodo and add them to a community, thus keeping track of all relevant
> data for an entire project. You can also track versions. Each dataset
> gets its own DOI. So you could e.g. cite your data management plan
> in the description field of your dataset.
{: .callout}
```

Comment (nielsdrost) [reply _episodes/documentation.md]: Zenodo is not introduced at all. Is this already discussed in a previous lesson?

Zenodo also allows multiple files for a single dataset. Changed the text, but perhaps it is too complex now.

File: _episodes/documentation.md [hunk 3]
```diff
@@ -1,33 +1,92 @@
...
+focus on more formalized metadata that is also, or even mainly, intended for
+machine readability.
 
-You can also log other important decisions that impact your data (sort of labjournal?!).
+## Exercises
 
-Documentation can take many forms: data management plan, papers published, (jupyter) notebooks, provencance information, etc. It is a good idea to bundle these with the data when you publish the data, so that people that find your data will not have to search for it elsewhere
+> ## Checking data documentation
+>
+> Visit a data repository of your choosing (e.g. hydroshare, 4TU, Zenodo, ...).
```
Comment (nielsdrost) [inline _episodes/documentation.md]: Should we include links to these repositories or is this part of the fun? ;-)

Comment (nielsdrost) [inline _episodes/documentation.md]:
```suggestion
> Visit a data repository of your choosing (e.g. HydroShare, 4TU.ResearchData, Zenodo, ...).
```

File: _episodes/documentation.md [hunk 4]
```diff
@@ -1,33 +1,92 @@
...
+>   project?
+> - Are contact details provided that you can use in case you have questions?
+> - Is there any description of the context in which this dataset was created?
+> - What could be improved about the documentation of this dataset?
+{: .challenge}
 
-{% include links.md %}
+> ## Writing your own data documentation
+>
+> For the real use case you selected in the previous episode, write a general
```
Comment (nielsdrost) [inline _episodes/documentation.md:82]: Perhaps we need to scope this somehow. "write a single page general description"?

File: _episodes/documentation.md [hunk 5]
```diff
@@ -1,33 +1,92 @@
...
+teaching: 5
+exercises: 15
 questions:
-- "Key question (FIXME)"
+- "What is documentation?"
+- "Where to document my data?"
+- "What is the difference between documentation and metadata?"
 objectives:
-- "First learning objective. (FIXME)"
+- "Evaluate existing data documentation with a critical view towards reusability"
```
Comment (nielsdrost) [inline _episodes/documentation.md:10]: Not sure this objective is covered in the lesson.

**Review by nielsdrost — CHANGES_REQUESTED** [review]
Nice work @Peter9192 and @stefsmeets!

I have a feeling this lesson could be a bit longer, but perhaps this is enough for now.

**Comment by Peter9192** [comment]
Hey @nielsdrost thanks for the review!

> I have a feeling this lesson could be a bit longer, but perhaps this is enough for now.

Is there anything specific that you're missing right now? I think the exercises will take up most time, which is intentional. I want to limit the time spent on one-way communication.

**Comment by nielsdrost** [comment]
> Hey @nielsdrost thanks for the review!
> 
> > I have a feeling this lesson could be a bit longer, but perhaps this is enough for now.
> 
> Is there anything specific that you're missing right now? I think the exercises will take up most time, which is intentional. I want to limit the time spent on one-way communication.

Well, specifically we are not explaining Zenodo in this lesson(but I guess it will be introduced somewhere else).

Also, there is very little on what makes for good documentation. Any examples? Information that should always be there?

But, agree that spending time on trying to create documentation is probably the most effective way to learn about documentation ;-)

File: _episodes/documentation.md (hunk 5 above)
Comment (Peter9192) [reply _episodes/documentation.md:10]: Well, it's one of the exercises. Perhaps we can add a checklist somewhere. I found a nice one [here](https://zenodo.org/record/3405141#.X32Tn-1R1hE) but maybe that one is better at the end of the lesson? I'll make an issue to add it somewhere (later).

Comment (Peter9192) [reply _episodes/documentation.md:10]: #20

File: _episodes/documentation.md (hunk 2 above)
Comment (Peter9192) [reply _episodes/documentation.md]: I like it, thanks!

File: _episodes/documentation.md (hunk 3 above)
Comment (Peter9192) [reply _episodes/documentation.md]: part of the fun I would say.

File: _episodes/documentation.md [hunk 6]
```diff
@@ -1,33 +1,92 @@
...
+>
+> For the real use case you selected in the previous episode, write a general
+> description of the dataset(s) that will be produced. This description may
+> include references to external documents. Also include the steps taken to
+> obtain the final result ('data flow'). Make sure it will be adequate for
+> others to understand how the data have been created. Also try to formulate a
+> suitable title for the dataset(s).
+>
+> You may also write down some more technical aspects of the data, such as
+> variable names, units, grids, creation date, software versions, etc.
```
Comment (Peter9192) [inline _episodes/documentation.md:90]:
```suggestion
> variable names, units, grids, creation date, software versions, etc.
>
> Limit your description to a single page (max).
```

File: _episodes/documentation.md (hunk 4 above)
Comment (Peter9192) [reply _episodes/documentation.md:82]: Added below.

**Comment by Peter9192** [comment]
@nielsdrost I addressed your comments (somehow my response have been interpreted as a new review..). Is it okay like this? Can you approve?

**Review by nielsdrost — APPROVED** [review]
Good work @stefsmeets and @Peter9192 !

**Pull request merged**